test/support/annotations/templates/header.erb

```diff
@@ -0,0 +1,7 @@
+---
+layout: plugins/katello/documentation
+title: Annotated Requests
+version: nightly
+foreman_version: nightly
+script: osmenu.js
+---
```

Comment:
The purpose is to be published on theforeman.org, so this is adding the page header in there for that.